scripts/devnet-cfx/Makefile

```diff
@@ -1,8 +1,9 @@
 IMAGE="reachsh/devnet-cfx"
+CONFLUX_VERSION="v1.1.0"
```

Comment:
We used to use 1.1.5?

Reply:
I have fixed the version to v1.1.5-testnet which is the one that we are already using.